plover/engine.py

```diff
@@ -321,6 +321,7 @@ def _on_machine_state_changed(self, machine_state):
 
     def _consume_engine_command(self, command):
         # The first commands can be used whether plover has output enabled or not.
+        original_command = command
```

Comment:
```suggestion
        command_name, command_args* = command
        # Lowercase command name but not the arguments.
        command_name = command_name.lower()
        command_args = command_args[0] if len(command_args) else ''
```

Reply:
This would need another string split (instead of string slice), but that's obviously fast enough.